hass_nabucasa/events/types.py

```diff
@@ -0,0 +1,41 @@
+"""Event types for cloud system."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from datetime import UTC, datetime
+from typing import TYPE_CHECKING, Literal
+
+if TYPE_CHECKING:
+    from hass_nabucasa.iot_base import DisconnectReason
```

Comment:
```suggestion
    from ..iot_base import DisconnectReason
```
Needs #1005 first as that adds exclusion in ruff for it.